### ISSUE TITLE
MySQL: Every derived table must have its own alias

### ISSUE THREAD

**Issue by filewalkwithme** [grafana/grafana#52569]
**What happened**:
Grafana is reporting the following error, which is generated by MySQL:
```
logger=login.authinfo.store t=2022-07-20T18:17:20.945527345Z level=error msg="Failed to get login stats" error="Error 1248: Every derived table must have its own alias"
logger=infra.usagestats t=2022-07-20T18:17:20.945560609Z level=error msg="Failed to fetch external metrics" error="Error 1248: Every derived table must have its own alias"
```

This error is being reported by at least two different loggers: 
- **login.authinfo.store** 
- **infra.usagestats**

When looking at [MySQL docs](https://dev.mysql.com/doc/mysql-errors/5.7/en/server-error-reference.html#error_er_derived_must_have_alias), we can find this:
```
Error number: 1248; Symbol: [ER_DERIVED_MUST_HAVE_ALIAS](https://dev.mysql.com/doc/mysql-errors/5.7/en/server-error-reference.html#error_er_derived_must_have_alias); SQLSTATE: 42000

Message: Every derived table must have its own alias
```

**What you expected to happen**:
No errors.

**How to reproduce it (as minimally and precisely as possible)**:
The error happens when running Grafana with a MySQL database.

**Anything else we need to know?**:
This error is happening because Grafana is trying to run a query which format is not recognized by MySQL

From a [StackOverflow](https://stackoverflow.com/questions/1888779/what-is-the-error-every-derived-table-must-have-its-own-alias-in-mysql) topic:
> This error is caused by the fact that you basically generate a new table with your subquery for the FROM command.
> That's what a derived table is, and as such, it needs to have an alias (actually a name reference to it).

- https://dev.mysql.com/doc/mysql-errors/5.7/en/server-error-reference.html#error_er_derived_must_have_alias
- https://stackoverflow.com/questions/1888779/what-is-the-error-every-derived-table-must-have-its-own-alias-in-mysql

I tried to run this query using sqlite and mysql, here are the results 
- sqlite:
![image](https://user-images.githubusercontent.com/1558179/180074320-a9188753-127b-40dd-ad3f-5f41d6861c8a.png)

- mysql
![image](https://user-images.githubusercontent.com/1558179/180074372-35caa922-0885-49a9-a6e5-e5f8ec7637dd.png)

**Environment**:
- Grafana version: 9.0.4
- Data source type & version: -
- OS Grafana is installed on: Ubuntu
- User OS & Browser: Ubuntu / Chrome
- Grafana plugins: - 
- Others: -

**Comment by eleijonmarck** [comment]
We are working on a fix that will be released for v9.0.5

**Comment by zoltanbedi** [comment]
I think we can close this then as this will be fixed and released in 9.0.5.

**Comment by filewalkwithme** [comment]
Thank you @eleijonmarck @zoltanbedi !!
